src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -91,8 +93,39 @@
                 {new File(ALLELE_SPECIFIC_DIRECTORY, "NA12878.AS.chr20snippet.g.vcf"), getTestFile( "AS_Annotations.gatk3.7_30_ga4f720357.expected.g.vcf"), Arrays.asList( "-A", "ClippingRankSumTest", "-G", "AS_StandardAnnotation", "-G", "StandardAnnotation"), b37_reference_20_21},
                 {getTestFile( "multiSamples.g.vcf"), getTestFile( "multiSamples.GATK3expected.g.vcf"), Arrays.asList( "-A", "ClippingRankSumTest", "-G", "AS_StandardAnnotation", "-G", "StandardAnnotation"), b37_reference_20_21},
                 {getTestFile( "testAlleleSpecificAnnotations.CombineGVCF.output.g.vcf"), getTestFile( "testAlleleSpecificAnnotations.CombineGVCF.expected.g.vcf"), Arrays.asList( "-A", "ClippingRankSumTest", "-G", "AS_StandardAnnotation", "-G", "StandardAnnotation"), b37_reference_20_21},
-                //all sites not supported yet see https://github.com/broadinstitute/gatk-protected/issues/580 and  https://github.com/broadinstitute/gatk/issues/2429
-                //{getTestFile(basePairGVCF), getTestFile( "gvcf.basepairResolution.includeNonVariantSites.gatk3.7_30_ga4f720357.expected.vcf"), Collections.singletonList("--"+GenotypeGVCFs.ALL_SITES_LONG_NAME) //allsites not supported yet
+
+                // all sites/--include-non-variant-sites tests
+
+                // The results from this test differ slightly from gatk3 since sites where the only alternate allele is a spanning
+                // deletion are not emitted by gatk4.
+                {getTestFile(BASE_PAIR_GVCF), getTestFile( "expected/gvcf.basepairResolution.includeNonVariantSites.vcf"), Collections.singletonList("--" + GenotypeGVCFs.ALL_SITES_LONG_NAME), b37_reference_20_21 },
+                {getTestFile( "combine.single.sample.pipeline.1.vcf"),
+                        getTestFile( "expected/combine.single.sample.pipeline.1.include_nonvariant.vcf"),
+                        Arrays.asList( "  --include-non-variant-sites -L 20:10,030,000-10,033,000 -L 20:10,386,000-10,386,500 "),
```

Comment:
I would use the constant here instead of hardcoding --include-non-variant-sites.  We're not consistent about it at all, but better to use the constants when adding new ones.  I think -L is so embedded in gatk that I wouldn't bother constantizing it...

Reply:
Good point - done.